src/implementations/vanilla/components/menu/_menu.scss

```diff
@@ -138,14 +138,6 @@ $_mobile-item-hover-background: null !default;
   }
 }
 
-/* stylelint-disable-next-line order/order */
-@include breakpoints.up('m') {
-  .ecl-menu {
```

Comment:
it seems that this spacing change is still needed, at least on EU:
![image](https://user-images.githubusercontent.com/25579221/118442176-8a127500-b6ea-11eb-92cf-e4cefc989b3f.png)
On EC, it seems to be the case too, even if that's not very clear...
![image](https://user-images.githubusercontent.com/25579221/118442280-b0d0ab80-b6ea-11eb-85a7-d6cce8e91882.png)

Reply:
Thks for review @emeryro, this was already validated by Erika in the comment of the ticket. Specs not being clear during implementation. After checking, Erika also found that the spacing for tablet was much too large, especially without a site name. Do you want to have this checked by the designers one last time?

Reply:
If you say that they validated, it is fine for me